From 0dffcc062bf7fab94d68cdab22bdc68ce7890d14 Mon Sep 17 00:00:00 2001
From: Yu Watanabe <watanabe.yu+github@gmail.com>
Date: Mon, 25 Jun 2018 14:07:16 +0900
Subject: [PATCH 06/25] install: fix error handling in
 is_symlink_with_known_name()

(cherry picked from commit d5c30911221634027f2800b8e887d164923ad2dd)
---
 src/shared/install.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/shared/install.c b/src/shared/install.c
index 77ae812878..564f3aec2a 100644
--- a/src/shared/install.c
+++ b/src/shared/install.c
@@ -681,7 +681,7 @@ static int remove_marked_symlinks(
         return r;
 }
 
-static bool is_symlink_with_known_name(const UnitFileInstallInfo *i, const char *name) {
+static int is_symlink_with_known_name(const UnitFileInstallInfo *i, const char *name) {
         int r;
 
         if (streq(name, i->name))
-- 
2.19.1

